avm/utilities/pipelines/staticValidation/psrule/.ps-rule/min-suppress.Rule.yaml

```
@@ -46,6 +46,7 @@ spec:
     # Azure API Management
     - Azure.APIM.MultiRegion # Team agreed this is too expensive for most use cases and is safe to ignore. Would require dependencies for a min deployment.
     - Azure.APIM.ManagedIdentity
+    - Azure.AKS.MaintenanceWindow # Excluded as it requires user input
```

Minor, just for consistency
```suggestion
    # AKS specific
    - Azure.AKS.MaintenanceWindow # Excluded as it requires user input
```